app/src/main/kotlin/com/k0d4black/theforce/commons/ExceptionHandler.kt

```diff
@@ -13,14 +13,20 @@
  **/
 package com.k0d4black.theforce.commons
 
+import android.app.Application
+import com.k0d4black.theforce.R
 import java.net.UnknownHostException
 
 object ExceptionHandler {
-     //TODO Load strings from resources for easier localization.
-    fun parse(t: Throwable): String {
+
+    fun parse(app: Application, t: Throwable): String {
```

Comment:
After the changes, this will return a string resource id . This will also require a change in the error state class that currently holds a message as a string. The context to get resources will be handled in the respective activities

Reply:
I realized it, I'll change it accordingly